cypress/features/regression/designSystem/flatTable.feature

```diff
@@ -1,22 +1,25 @@
 Feature: Design Systems FlatTable component
-  I want to test Design Systems FlatTable component properties
+  I want to test Design Systems FlatTable component
 
   Background: Open Design Systems FlatTable component page
     Given I open Design Systems basic "Flat Table" component docs page
 
   @positive
   Scenario: FlatTable has sticky row
+    # commented because of BDD default scenario Given - When - Then
     Then FlatTable rows are sticky
```

Comment:
What commented? :) `When` needs also to be added :)

Reply:
fixed